admin_guide/diagnostics_tool.adoc

```diff
@@ -298,6 +302,24 @@ A user-defined timeout may be set by passing the
 if a newly-created log entry is not able to be queried via Elasticsearch after
 45 seconds.
 
+|`sdn`
+a|This check performs the following cluster-level diagnostics of the
+{product-title} SDN:
+
+* Verify that master hosts can connect to kubelets.
+* Verify that nodes can route packets to each other.
+* Validate node addresses.
+* Validate HostSubnet objects.
+
+If you specify the `*openshift_checks_output_dir*` variable, the check also
```

Comment:
Where to specify `openshift_checks_output_dir`?

Reply:
I added a sentence referring to the example usage below.  Is that sufficient, or would it be preferable to add an explicit example of setting `openshift_checks_output_dir` in the usage examples?

Reply:
@Miciah It is not clear to me where the user sets the variable.

Reply:
Usually the user would set the variable when running the `ansible-playbook` command.  (The user could specify it in the inventory file, but it probably is not helpful to mention that.)

Would it help to move the "see also" into the first sentence? "If you specify the `*openshift_checks_output_dir*` variable (see example usage of the `ansible-playbook` command below) [...]"

Or should I replace or supplement the "see also" at the end of the paragraph with something more explicit? "For example, use `ansible-playbook -i <inventory_file> ~/openshift-ansible/playbooks/openshift-checks/health.yml -e openshift_checks=sdn -e openshift_checks_output_dir=./ansible-output`.  See example usage of the `ansible-playbook` command below."

Reply:
Miciah. I didn't realize that you add the variable when you run the playbook. maybe:
_If you specify the `*openshift_checks_output_dir*` variable with the `ansible-playbook` command..._

Reply:
All right, I've added "with the `ansible-playbook` command".  Thanks!